src/elements/fields/index.jsx

```diff
@@ -191,7 +191,8 @@ function applyFieldStyles(field, styles) {
       break;
     default:
       styles.addTargets('sub-fc', 'tooltipIcon');
-      if (type !== 'checkbox') styles.applyWidth('fc'); // Avoid applying width to checkbox to ensure the checkbox width is properly set by the component
+      // Avoid applying width to checkbox to ensure the checkbox width is properly set by the component
```

Comment:
the autoformat kept moving this comment from the end of the line to be above the `default:` line, so I moved it to be in front of the appropriate line

Reply:
can you move it right above the line? instead of right below it